src/pepr/values.yaml

```diff
@@ -4,12 +4,12 @@
 watcher:
   serviceMonitor:
     enabled: ###ZARF_VAR_PEPR_SERVICE_MONITORS###
-  envFrom:
-    - secretRef:
-        name: uds-operator-config
+  env:
+    - name: UDS_LOG_LEVEL
+      value: ###ZARF_VAR_UDS_LOG_LEVEL###
```

Comment:
I think this loses two values/overrides we still want:
- PEPR_RECONCILE_STRATEGY
- PEPR_KEYCLOAK_CLIENT_STRATEGY

I'm not seeing another place/way those are loaded? We could hardcode `PEPR_RECONCILE_STRATEGY` if we needed/wanted to. `PEPR_KEYCLOAK_CLIENT_STRATEGY` is a new one, but should be overridable by the end user. Not sure if it needs to be part of the clusterconfig because ideally it's temporary but curious on @slaskawi opinion?

Reply:
Yes, I agree - there should be an easy way to override these. Right now they are defined here:

https://github.com/defenseunicorns/uds-core/blob/05def890049a4cb33e19634f93f08bed008cb09f/src/pepr/uds-operator-config/values.yaml#L20-L21

Just for the record, once the Client Credentials proves to run without any problems, the Dynamic Client Registration will be deprecated and potentially removed.

Reply:
`PEPR_KEYCLOAK_CLIENT_STRATEGY` can be removed now I think, since we deprecated the options for that.